test/distributed/org/apache/cassandra/fuzz/topology/TopologyMixupTestBase.java

```diff
@@ -400,16 +422,20 @@ protected static class State<S extends SchemaSpec> implements AutoCloseable
         final TopologyHistory topologyHistory;
         final Cluster cluster;
         final S schemaSpec;
+        final List<BiFunction<State<S>, Gen<Command<State<S>, Void, ?>>, Gen<Command<State<S>, Void, ?>>>> commandsTransformers = new ArrayList<>();
```

Comment:
Nit: should we maybe alias this to something? Like an interface, maybe Command Generator and a named function.